src/database/models/index.js

```diff
@@ -8,19 +9,40 @@ import envConfigs from "../config/config";
 dotenv.config();
 const basename = _basename(__filename);
 const env = process.env.NODE_ENV;
+=======
```

Comment:
Check this file as they're many unwanted lines and characters that may have come after rebasing
```suggestion
```